### PR TITLE
Add Symfony Armenian Translations

### LINKED ISSUE #40204: Missing translations for Armenian (hy)
Hello,

There are some missing translation for Armenian (hy). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.hy.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Form/Resources/translations/validators.hy.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by ErikSaunier**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #40204
| License       | MIT
| Doc PR        | 

Added missing armenian translations for Form components

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you for the PR. 

@azatyan could you please verify this?

**Comment by ErikSaunier** [comment]
Im happy with this PR. But I would be happy if a native speaker double checked.

File: src/Symfony/Component/Form/Resources/translations/validators.hy.xlf [hunk 1]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter a valid birthdate.</source>
+                <target>Խնդրում ենք մուտքագրել վավեր ծննդյան ամսաթիվ։</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>Ընտրված ընտրությունն անվավեր է։</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>Հավաքածուն անվավեր է։</target>
```
Comment (azatyan) [inline src/Symfony/Component/Form/Resources/translations/validators.hy.xlf]: Համախումբն անվավեր է (it's more mathemathical than «հավաքածու»)

File: src/Symfony/Component/Form/Resources/translations/validators.hy.xlf [hunk 2]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please select a valid country.</source>
+                <target>Խնդրում ենք ընտրել վավեր երկիր։</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Խնդրում ենք ընտրել վավեր արժույթ։</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Խնդրում ենք ընտրել ճիշտ ամսաթվի ընդմիջում։</target>
```
Comment (azatyan) [inline src/Symfony/Component/Form/Resources/translations/validators.hy.xlf]: ամսաթվերի միջակայք

File: src/Symfony/Component/Form/Resources/translations/validators.hy.xlf [hunk 3]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please provide a valid phone number.</source>
+                <target>Խնդրում ենք տրամադրել վավեր հեռախոսահամար։</target>
+            </trans-unit>
+            <trans-unit id="124">
+                <source>The checkbox has an invalid value.</source>
+                <target>Նշման վանդակը անվավեր արժեք ունի։</target>
+            </trans-unit>
+            <trans-unit id="125">
+                <source>Please enter a valid email address.</source>
+                <target>Խնդրում ենք մուտքագրել վավերական էլեկտրոնային հասցե։</target>
```
Comment (azatyan) [inline src/Symfony/Component/Form/Resources/translations/validators.hy.xlf]: վավեր էլ-հասցե || վավեր էլ-փոստ

**Review by azatyan — COMMENTED** [review]
In general all is okay, but with small bugs, you can fix @ErikSaunier

**Comment by Nyholm** [comment]
Thank you @azatyan. Im happy with this PR after @ErikSaunier have responded to the suggestions.

**Comment by ErikSaunier** [comment]
Thank you @azatyan for your help. Do my modifications seem correct to you?

**Review by javiereguiluz — APPROVED** [review]

**Review by azatyan — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ErikSaunier.

**Pull request merged**